modules/arrow/src/geoarrow/convert-geojson-feature-to-arrow.ts

```diff
@@ -0,0 +1,375 @@
+import {
+  Float64 as ArrowFloat,
+  FixedSizeList,
+  Field as ArrowField,
+  List as ArrowList,
+  Vector as ArrowVector,
+  makeBuilder
+} from 'apache-arrow';
+import {
+  Point,
+  MultiPoint,
+  LineString,
+  MultiLineString,
+  Polygon,
+  MultiPolygon,
+  Feature,
+  GeoJsonObject
+} from 'geojson';
+
+/**
+ * Arrow Field and geometry vector
+ */
+export type GeoArrowReturnType = {
```

Comment:
Seems more general, overall I would like us to have support for all column types not just GeoArrow geometry:
```suggestion
export type ArrowColumn = {
```